compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -3090,7 +3090,7 @@ OMR::Power::CodeGenerator::loadAddressConstantFixed(
       bool doAOTRelocation)
    {
    TR::Compilation *comp = self()->comp();
-   bool isAOT = comp->compileRelocatableCode();
+   bool isAOT = comp->compileRelocatableCode() || comp->isOutOfProcessCompilation();
```

Comment:
Be sure to change this `isAOT` variable's name to something more appropriate when you have the new query.

Reply:
Variable name changes here: 200634121c31bd0a24e7130ee36375fa8e3fb34e